save_preferences/save_preferences.js

```diff
@@ -1,18 +1,19 @@
-var allRanges = document.querySelectorAll(".range-wrap");
-allRanges.forEach(function (wrap) {
-    var range = wrap.querySelector(".range");
-    var bubble = wrap.querySelector(".bubble");
-    range.addEventListener("input", function () {
+"use strict";
+const allRanges = document.querySelectorAll(".range-wrap");
+allRanges.forEach(wrap => {
+    const range = wrap.querySelector(".range");
+    const bubble = wrap.querySelector(".bubble");
+    range.addEventListener("input", () => {
         setBubble(range, bubble);
     });
     setBubble(range, bubble);
 });
 function setBubble(range, bubble) {
-    var val = range.value;
-    var min = range.min ? range.min : 0;
-    var max = range.max ? range.max : 100;
-    var newVal = Number(((val - min) * 100) / (max - min));
+    const val = range.value;
+    const min = range.min ? range.min : 0;
+    const max = range.max ? range.max : 100;
+    const newVal = Number(((val - min) * 100) / (max - min));
     bubble.innerHTML = val;
     document.getElementById("calories").value = val;
-    bubble.style.left = "calc(" + newVal + "% + (" + (8 - newVal * 0.15) + "px))";
+    bubble.style.left = `calc(${newVal}% + (${8 - newVal * 0.15}px))`;
```

Comment:
Why did this change? Have you compiled my TS files again with your TS compiler?

Reply:
Yes the compiler does these changes.